src/workloads/execution/SlidingWindows.yml

```diff
@@ -10,11 +10,11 @@ Actors:
     Database: &db test
     Threads: 1
     CollectionCount: 1
-    DocumentCount: 100000
-    BatchSize: 10000
+    DocumentCount: 10000
+    BatchSize: &batchSize 10000
     Document:
-      part: {^RandomInt: {min: 1, max: 10}}
-      time: {^RandomDate: {min: 1588626902000, max: 1620162902000}}
+      partitionKey: {^RandomInt: {min: 1, max: 5}}
```

Comment:
I'm surprised you needed to change this, since having less partitions conceptually means that each partition has more documents and is more likely to spill. But if it works then keep it!